templates/mysql.type.go.tpl

```diff
@@ -98,15 +98,27 @@ func ({{ $short }} *{{ .Name }}) Insert(db XODB) error {
 			return errors.New("update failed: marked for deletion")
 		}
 
-		// sql query
-		const sqlstr = `UPDATE {{ $table }} SET ` +
-			`{{ colnamesquery .Fields ", " .PrimaryKey.Name }}` +
-			` WHERE {{ colname .PrimaryKey.Col }} = ?`
-
-		// run query
-		XOLog(sqlstr, {{ fieldnames .Fields $short .PrimaryKey.Name }}, {{ $short }}.{{ .PrimaryKey.Name }})
-		_, err = db.Exec(sqlstr, {{ fieldnames .Fields $short .PrimaryKey.Name }}, {{ $short }}.{{ .PrimaryKey.Name }})
-		return err
+		{{ if gt ( len .PrimaryKeyFields ) 1 }}
+			// sql query with multiple PK's
```

Comment:
Good point thanks, I was being a little loose with the terminology there, will fix it later.